requirements.txt

```diff
@@ -1,10 +1,8 @@
-PyChromecast==0.8.2
-soco==0.12
-Flask==0.12.2
-netifaces==0.10.6
-setuptools==36.6.0
-requests==2.18.4
-mutagen==1.38
-psutil==5.4.0
+PyGObject
```

Comment:
I think this dependency creates problems.

Reply:
This is the result of merging commit: 47ca6a1ad941916965cac456922093a89f0b9c19. 

I have build it using `debuild -b -uc -us -tc` no obvious error was shown. What do you recommend how to change it? I have no clue. I don't know much about this file and python. If the change creates problems, shouldn't it be changed then in the master? Or will it only create problems in the debian branch?

Reply:
It will only create problems when installing the dependencies with pip. But it is not a big deal right now. I would just remove it from the `requirements.txt`.

Reply:
Okay that should be done then on master branch and is not relevant for this PR, correct?